website/docs/getting-started/setup/instance-management/appsmithctl.mdx

```diff
@@ -39,64 +39,77 @@ Subcommands allow you to trigger different operations like exporting or importin
 You can create a backup archive of the Appsmith instance. The backup includes the database, `docker.env`, and Git data. Follow these steps to create a backup archive:
 
 <Tabs queryString="current-command-type">
-<TabItem label="Docker" value="docker-commands"> 
+<TabItem label="Docker" value="docker-commands">
 
 1. Go to the directory that has the `docker-compose.yml` file.
 
-2. Get your encryption details with:
+2. You may choose to create a backup archive with or without encryption.
 
-```bash
-docker-compose exec appsmith cat /appsmith-stacks/configuration/docker.env | grep ENCRYPTION
-```
-Keep these details handy, as you need them for restoring the Appsmith instance.
+    * **With encryption** - To create a backup archive with encryption, execute the following command:
 
-3. Create a backup archive with:
+      ```bash
+      docker-compose exec appsmith appsmithctl backup
+      ```
 
-```bash
-docker-compose exec appsmith appsmithctl backup
-```
-The backup archive file is stored in the `/appsmith-stacks/data/backup/` folder in your container directory. You can access it at `./stacks/data/backup/` on your local machine.
+      The above command encrypts the backup archive with a password. When prompted, enter the password and ensure you remember it. You need this password if you ever want to restore from this backup. If you forget it, you won't be able to restore this backup.
 
-4. If you can't access or have a different volume configuration, copy the archive file to your host disk with:
+    * **Without encryption** - To create a backup archive without encryption, execute the following command:
 
-```bash
-# Replace the appsmith-backup-DATE_AND_TIMESTAMP.tar.gz with the backup file name
-docker cp appsmith:/appsmith-stacks/data/backup/appsmith-backup-DATE_AND_TIMESTAMP.tar.gz .
-```
-After running the above command, the backup archive is copied to your current directory, indicating that the backup process is completed. You can now proceed with the remaining steps of the process you were performing.
+      ```bash
+      docker-compose exec appsmith appsmithctl backup `--non-interactive`
+      ```
+
+      The above command creates a backup without encryption, and the backup archive won't include `APPSMITH_ENCRYPTION_PASSWORD` and `APPSMITH_ENCRYPTION_SALT` environment variables in the `docker.env` file. Make sure to note them separately if you choose this option.
+
+    The backup archive file is stored in the `/appsmith-stacks/data/backup/` folder in your container directory. You can access it at `./stacks/data/backup/` on your local machine.
+
+3. If you can't access or have a different volume configuration, copy the archive file to your host disk with:
+
+   ```bash
+   # Replace the appsmith-backup-DATE_AND_TIMESTAMP.tar.gz with the backup file name
+   docker cp appsmith:/appsmith-stacks/data/backup/appsmith-backup-DATE_AND_TIMESTAMP.tar.gz .
```

Comment:
Encrypted backups end with `.tar.gz.enc`. Should we mention that as well?

Reply:
Is the above `appsmith-backup-DATE_AND_TIMESTAMP.tar.gz` not correct?